Library/Formula/byobu.rb

```diff
@@ -19,6 +20,9 @@ class Byobu < Formula
   depends_on "tmux"
   depends_on "newt" => "with-python"
 
+  depends_on "automake" => :build
```

Comment:
@cosmo0920 Yeah, this should be conditional. Should be something like:

```ruby
head do
  url "https://github.com/dustinkirkland/byobu.git"

  depends_on "automake" => :build
  depends_on "autoconf" => :build
end
```

It's likely you'll also need a:

```ruby
system "autoreconf", "-fvi" if build.head?
```
Line above `./configure` in the install stage, if you require the autotool deps.